### PR TITLE
Added missing status messages.

### LINKED ISSUE #174: Not knowing immediately if verification started again or not.
When I modify my code with verification on change, the status bar is not updated until the code actually is resolved or parsing or resolution failed.
This is misleading, because for large file, parsing and resolution can take a substantial time (a few seconds), during which it will still display the previous "Verifying... "message, which leads me to think that Dafny is stuck verifying since I changed the code and it does not update the status bar immediately.
We should reflect immediately that Dafny took the change into account and is processing it.

### PR DESCRIPTION
**Pull request by MikaelMayer**
Fixes #174 but will need Dafny Language Server to support those messages afterwards.

Alternative to discuss:
* Should we replace "Parsing started" with "Resolution started" and remove the second "Resolution started" message?

### CODE REVIEW

**Comment by keyboardDrummer** [comment]
> Should we replace "Parsing started" with "Resolution started" and remove the second "Resolution started" message?

I think that's better. The user doesn't benefit from each having finished only parsing. It doesn't enable any additional IDE features.

**Comment by MikaelMayer** [comment]
> I think that's better

Done. I also changed the message from "Resolution Started" to "Resolving..." which is more consistent with "Verifying..."

File: src/ui/compilationStatusView.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ const StatusBarPriority = 10;
 
 function toStatusMessage(status: CompilationStatus, message?: string | null): string {
   switch(status) {
+  case CompilationStatus.ResolutionStarted:
```
Comment (keyboardDrummer) [inline src/ui/compilationStatusView.ts:12]: Shouldn't this be when it starts parsing already?

Comment (MikaelMayer) [reply src/ui/compilationStatusView.ts:12]: Yes, if resolution for the user is parsing + internal resolution, then resolution started is what we report.

**Pull request merged**